src/olympia/devhub/templates/devhub/new-landing/index.html

```diff
@@ -1 +1,64 @@
-New DevHub Landing
+<!DOCTYPE html>
+<html lang="{{ LANG }}" dir="{{ DIR }}">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% if not settings.ENGAGE_ROBOTS %}
+      <meta name="robots" content="noindex">
+    {% endif %}
+    {% block extrameta %}{% endblock %}
+
+    <title>{% block title %}{{ dev_page_title() }}{% endblock %}</title>
+
+    <link rel="shortcut icon" type="image/x-icon"
+          href="{{ STATIC_URL }}img/favicon.ico">
+
+    {% block rss_feed %}{% endblock %}
+
+    {# L10n: {0} is an application, like Firefox. #}
+    <link title="{{ _('{0} Add-ons')|f(APP.pretty) }}"
+          rel="search" type="application/opensearchdescription+xml"
+          href="{{ url('amo.opensearch') }}" />
+
+    {% block site_css %}
+      {{ css('devhub/css') }}
+    {% endblock %}
+
+    {% block extrahead %}{% endblock %}
+
+    {% if request.user.is_authenticated() %}
+      <meta name="csrf" content="{{ csrf_token }}">
+    {% endif %}
+
+    {{ js('preload') }}
+  </head>
+  <body class="html-{{ DIR }} {{ request.APP.short }} {% block bodyclass %}{% endblock %}"
+        data-raven-dsn="{{ settings.RAVEN_DSN }}"
+        data-raven-urls="{{ settings.RAVEN_ALLOW_LIST }}"
+        {% block bodyattrs %}{% endblock %}>
+
+    <nav class="DevHub-Navigation">
+      <ul>
+        <li><a href="#"><div class="Logo" alt="{{ _('Mozilla Firefox') }}" /></a></li>
+        <li><a href="#">Docs</a></li>
+        <li><a href="#">Support</a></li>
+        <li><a href="#">Blog</a></li>
+      </ul>
+    </nav>
+
+    <section class="DevHub-Header">
+      <h1>{{ _('Customize Your Firefox') }}</h1>
+      <p>
+        {% trans %}
+        Add-ons let millions of Firefox users enhance their browsing experience.
+        If you know <a href="#">HTML</a>, <a href="#">JavaScript</a>, and <a href="#">CSS</a>,
```

Comment:
I'd delay that to a follow-up, I might redo how things are translated here and there anyway once I extract things and see how they look from a translators perspective. There's lots more translatable content to go so I'll do that a bit later.